tests/testthat/_snaps/rows-dm.md

```diff
@@ -29,13 +29,20 @@
       out <- dm_rows_insert(flights_sqlite, flights_hour10_sqlite)
     Message <simpleMessage>
       Not persisting, use `in_place = FALSE` to turn off this message.
+    Message <dplyr_message_matching_by>
```

Comment:
We shouldn't see those.

Reply:
I used the same class for consistency even though it doesn't really fit the conventions regarding packages. What exactly do you want:

a) messages without a class?
b) messages with a different class, e.g. `c("dm_message_matching_by", "dm_message")`?

Reply:
No message.